content/report_RHF.tex

```diff
@@ -1,9 +1,244 @@
+\end{equation}
+Off course the same argument stands for the $\beta$-spins.
+
+Now we have come to a point where we can define a restricted closed shell system.
+Imagine what would happen if all orbitals were doubly occupied, with an $\alpha$
+and a $\beta$ electron, all according to the Pauli princple. This has some consequences.
+The first one is that the amount of $\alpha$-electrons would equal the amount of
+$\beta$-electrons. Secondly, this would mean that the difference between these electrons
+is only their spin function. They reside in the same orbitals. This means that we only
+need to account for half the electrons, since the other half is in the same orbitals,
+just with a different spin. Now think about the Fock operator. Let us
+take a look at a single electron. It has a one election hamiltonian, electronic repulsion
+for all other electrons in the system and it only "exchanges" with half of the other
+electrons, only those with the same spin. This means we have a double the amount of 
+coulomb operators to exchange operators. When we now look back at Equation \ref{eq: fock matrix},
+we see that we are finally there.
+
+\subsection{Application of the Hartree-Fock Theory}
+\label{subsec:applic}
+At this point, we might think back to the point where we defined the coulomb and
+exchange operators. These operators themselves depend on the orbitals. So we would
+need to know the orbitals to define our operator. It seems we are stuck again.
+However we can apply some mathemathical tricks to get us out of this impasse. 
+First we can assert that every function can be expressed as a linear combination
+of basis functions. We also know that the eigenfunctions of a Hermitian operator
+form such a set of basis functions. We begin by selecting a convenient basis set.
+This could be a set of atomic orbitals. Then we would solve the eigenproblem in
+Equation \ref{eq:eigenproblem}.
+
+\begin{equation}\label{eq:eigenproblem}
+    \boldsymbol{\hat{H}}^{core}\boldsymbol{C} = \boldsymbol{SC}c
+\end{equation} 
+Calculating the eigenvalues and eigenfunctions for the core Hamiltonian is 
+perfectly possible, since it is a one-electron operator. We now have a set of 
+orbitals which we can use to build our Fock operator. For this Fock operator, 
+we can solve the generalised eigenproblem in the form of the Roothaan-Hall 
+equations (see Equation \ref{eq:Roothaan-Hall}).
+
+\begin{equation}\label{eq:Roothaan-Hall}
+    \boldsymbol{FC} = \boldsymbol{SC}\epsilon
+\end{equation}
+
+Solving this equation will give us a set of eigenfunctions of the Fock matrix. 
+These functions can then be used to build a new Fock matrix. This is where we 
+start the iterations. We can continue untill we reach a point where the energy 
+does not change any more. At that point, we have found the energy of the molecule.
+
+Before we move on, some clarifications are in order. 
+\begin{enumerate}
+    \item The basis set we choose is not the same as the set that is derived

```

Comment:
This block is not entirely accurate, but i admit that my comment on the previous version was maybe not entirely clear as well. Better delete this block for now and ask me in office about it. You can then decide to add a modified version again.